arches/management/commands/validate.py

```diff
@@ -0,0 +1,120 @@
+"""
+ARCHES - a program developed to inventory and manage immovable cultural heritage.
+Copyright (C) 2013 J. Paul Getty Trust and World Monuments Fund
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from datetime import datetime
+
+from arches import __version__
+from arches.app.const import IntegrityCheck
+from arches.app.models import models
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from django.db.models import Exists, OuterRef
+
+DELETE_QUERYSET = "delete queryset"
+
+
+class Command(BaseCommand):
+    """
+    Validate an Arches database against a set of data integrity checks.
+    Takes no action by default (other than printing a summary).
+
+    Provide --verbosity=2 to get a richer output (list of affected rows).
+
+    Example: python manage.py validate --fix-all
+    """
+    help = "Validate an Arches database against a set of data integrity checks, and opt-in to remediation."
+
+    def add_arguments(self, parser):
```

Comment:
running just the bare command w/o any flags throws an error
eg: `python manage.py validate`

Reply:
Ouch, that's due to some hasty last minute changes, will fix 👍